### PR TITLE
Added AVX swish/lrn/batchnorm 

### PR DESCRIPTION
**Pull request by Timen**
Added the AVX version of swish lrn and batchnorm.

And increase convolution test coverage + unrolled some more loops.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1897) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1897) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Timen<br/>:x: restyled-commits<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=1897) it.</sub>

File: tests/test_convolution.cpp [hunk 1]
```diff
@@ -257,5 +259,5 @@ static int test_convolution_1()
 int main()
 {
     SRAND(7767517);
-    return test_convolution_0() || test_convolution_1() || test_convolution_2();
+    return test_convolution_2() || test_convolution_1() || test_convolution_0();
```
Comment (nihui) [inline tests/test_convolution.cpp:262]: order reversed  :smile:

**Comment by nihui** [comment]
looks good to me

File: tests/test_convolution.cpp (hunk 1 above)
Comment (Timen) [reply tests/test_convolution.cpp:262]: I know but otherwise it fails🤭

Comment (Timen) [reply tests/test_convolution.cpp:262]: I suspect some alllocator issue, I will investigate more :) It was weird as it didn't fail on the ctest coverage I added but only a couple of subtests later. And it is fixed as I said by switching the order around 🤷‍♂️

Comment (Timen) [reply tests/test_convolution.cpp]: Undid the re-order and actually fixed the issue :)

**Comment by Timen** [comment]
I managed to track down the issue with the failing ctest to the sigmoid function producing nans. It seems that a specific set of randomly generated weight and input matrices would cause a very large exponent (>80) which the `exp()` function would return as a nan. 

The fix was to simply reduce the float range of the randomly generated matrices from -2/2 to -1.5/1.5.

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**